--- make.conf	2006-03-19 18:40:11.000000000 +0100
+++ make.conf.alpha	2006-03-19 18:26:21.000000000 +0100
@@ -23,6 +23,17 @@
 # Example:
 #USE="X gtk gnome -alsa"
 
+# Host Setting
+# ============
+#
+# DO NOT CHANGE THIS SETTING UNLESS YOU ARE USING STAGE1!
+# The generic HOST setting on alpha is alpha-unknown-linux-gnu. If your machine
+# is an ev6 or ev67 based system you might want to use 
+# either alphaev6-unknown-linux-gnu or alphaev67-unknown-linux-gnu accordingly.
+#
+#CHOST="alphaev67-unknown-linux-gnu"
+CHOST="alpha-unknown-linux-gnu"
+
 # Host and optimization settings 
 # ==============================
 #
@@ -33,10 +44,18 @@
 # package (and in some cases the libraries it uses) at default optimizations
 # before reporting errors to developers.
 #
-# Please refer to the GCC manual for a list of possible values.
+# -mcpu=<cpu-type> means optimize code for the particular type of CPU. In 
+#                  difference to x86 for example -mcpu does break compatibility
+#                  to older cpu types in case of ev6 or higher.
+#                  On Alpha there is no -march= option in gcc-3.
 #
-#CFLAGS="-O2 -pipe"
+# CPU types supported in gcc-3.2 or higher: ev4, ev45, ev5, ev56, ev6, ev67
 #
+# Decent examples:
+#
+#CFLAGS="-mcpu=ev67 -O2 -pipe "
+CFLAGS="-mcpu=ev5 -O2 -pipe "
+
 # If you set a CFLAGS above, then this line will set your default C++ flags to
 # the same settings.
 #CXXFLAGS="${CFLAGS}"
@@ -61,7 +80,7 @@
 # DO NOT PUT ANYTHING BUT YOUR SPECIFIC ~ARCHITECTURE IN THE LIST.
 # IF YOU ARE UNSURE OF YOUR ARCH, OR THE IMPLICATIONS, DO NOT MODIFY THIS.
 #
-#ACCEPT_KEYWORDS="~arch"
+#ACCEPT_KEYWORDS="~alpha"
 
 # Portage Directories
 # ===================
