--- make.conf.example
+++ make.conf.example
@@ -22,6 +22,13 @@
 # Example:
 #USE="X gtk gnome -alsa"
 
+# Host Setting
+# ============
+#
+# All MIPS systems should use this host setting:
+
+CHOST="mips-unknown-linux-gnu"
+
 # Host and optimization settings
 # ==============================
 #
@@ -39,10 +46,15 @@
 # -frecord-gcc-switches, since otherwise the check could result in false
 # positive results.
 #
-# Please refer to the GCC manual for a list of possible values.
+# -mcpu=<cpu-type> for MIPS systems selects the type of processor you want
+# to optimize your code for.   Code generated under those options will run best
+# on that processor, and may not run at all on others.
 #
-#CFLAGS="-O2 -pipe"
+# GCC 3.2 supports many mips processor types including: r2000, r3000, r3900, r4000, 
+# r4100, r4300, r4400, r4600, r4650, r5000, r6000, r8000 , orion
 #
+#CFLAGS="-O2 -pipe"
+
 # If you set a CFLAGS above, then this line will set your default C++ flags to
 # the same settings.
 #CXXFLAGS="${CFLAGS}"
@@ -76,7 +88,7 @@
 # DO NOT PUT ANYTHING BUT YOUR SPECIFIC ~ARCHITECTURE IN THE LIST.
 # IF YOU ARE UNSURE OF YOUR ARCH, OR THE IMPLICATIONS, DO NOT MODIFY THIS.
 #
-#ACCEPT_KEYWORDS="~arch"
+#ACCEPT_KEYWORDS="~mips"
 
 # ACCEPT_LICENSE is used to mask packages based on licensing restrictions.
 # It may contain both license and group names, where group names are
