--- make.conf.example
+++ make.conf.example
@@ -22,6 +22,15 @@
 # Example:
 #USE="X gtk gnome -alsa"
 
+# Host Setting
+# ============
+# This should be left to the default value of sparc-unknown-linux-gnu unless
+# you are absolutely certain of the consequences.  In addition to potentially
+# destroying your system, you will receive no support and your bugs will be
+# marked INVALID if you change this.
+#
+# CHOST="sparc-unknown-linux-gnu"
+
 # Host and optimization settings
 # ==============================
 #
@@ -39,10 +48,34 @@
 # -frecord-gcc-switches, since otherwise the check could result in false
 # positive results.
 #
-# Please refer to the GCC manual for a list of possible values.
-#
-#CFLAGS="-O2 -pipe"
-#
+# Sparc specific cpu optimizatiobn flags can be found here:
+# https://gcc.gnu.org/onlinedocs/gcc-4.9.3/gcc/SPARC-Options.html
+#
+# -mcpu=<cpu-type> means optimize code for the particular type of CPU without
+# breaking compatibility with other CPUs.
+#
+# -mtune=<cpu-type> set the instruction scheduling parameters for machine, but
+# do not set the instruction set or register set.
+#
+# Here is a list of each supported architecture and their supported
+# implementations:
+#
+# gcc-3.2 and higher:
+#    v7:             cypress
+#    v8:             supersparc, hypersparc
+#    sparclite:      f930, f934, sparclite86x
+#    sparclet:       tsc701
+#    v9:             ultrasparc
+#
+# Additionally in gcc-3.3 and higher:
+#    v9:             ultrasparc3
+#
+# Decent examples:
+#
+#CFLAGS="-mcpu=supersparc -O2 -pipe"
+#CFLAGS="-mcpu=ultrasparc -O2 -pipe"
+#CFLAGS="-mcpu=v8 -mtune=v9 -O2 -pipe"
+
 # If you set a CFLAGS above, then this line will set your default C++ flags to
 # the same settings.
 #CXXFLAGS="${CFLAGS}"
@@ -76,7 +109,7 @@
 # DO NOT PUT ANYTHING BUT YOUR SPECIFIC ~ARCHITECTURE IN THE LIST.
 # IF YOU ARE UNSURE OF YOUR ARCH, OR THE IMPLICATIONS, DO NOT MODIFY THIS.
 #
-#ACCEPT_KEYWORDS="~arch"
+#ACCEPT_KEYWORDS="~sparc"
 
 # ACCEPT_LICENSE is used to mask packages based on licensing restrictions.
 # It may contain both license and group names, where group names are
