--- make.conf.example
+++ make.conf.example
@@ -22,6 +22,13 @@
 # Example:
 #USE="X gtk gnome -alsa"
 
+# Host Setting
+# ============
+#
+# m68k-unknown-linux-gnu for all m68k CPUs.
+#
+CHOST="m68k-unknown-linux-gnu"
+
 # Host and optimization settings
 # ==============================
 #
@@ -41,7 +48,7 @@
 #
 # Please refer to the GCC manual for a list of possible values.
 #
-#CFLAGS="-O2 -pipe"
+#CFLAGS="-m68030 -pipe -Os"
 #
 # If you set a CFLAGS above, then this line will set your default C++ flags to
 # the same settings.
@@ -76,7 +83,7 @@
 # DO NOT PUT ANYTHING BUT YOUR SPECIFIC ~ARCHITECTURE IN THE LIST.
 # IF YOU ARE UNSURE OF YOUR ARCH, OR THE IMPLICATIONS, DO NOT MODIFY THIS.
 #
-#ACCEPT_KEYWORDS="~arch"
+#ACCEPT_KEYWORDS="~m68k"
 
 # ACCEPT_LICENSE is used to mask packages based on licensing restrictions.
 # It may contain both license and group names, where group names are
