--- make.conf.example
+++ make.conf.example
@@ -22,6 +22,19 @@
 # Example:
 #USE="X gtk gnome -alsa"
 
+# Host Setting
+# ============
+#
+# DO NOT CHANGE THIS SETTING UNLESS YOU ARE USING STAGE1!
+# It's very important you select the right CHOST from the start.  A wrong
+# CHOST could easily lead to weird errors either in compiling or running.
+#
+# There are really only two choices right now:
+#   sh3-unknown-linux-gnu
+#   sh4-unknown-linux-gnu    (Dreamcast / LANTank)
+#
+CHOST="sh4-unknown-linux-gnu"
+
 # Host and optimization settings
 # ==============================
 #
@@ -39,10 +52,15 @@
 # -frecord-gcc-switches, since otherwise the check could result in false
 # positive results.
 #
-# Please refer to the GCC manual for a list of possible values.
+# -m# optimize code for the particular type of CPU.  The number should match
+# your CHOST so if you are using "sh4-unknown-linux-gnu", you should have
+# -m4 below.
 #
-#CFLAGS="-O2 -pipe"
+# For a full listing of supported CPU models, please refer to the GCC website:
+# https://gcc.gnu.org/onlinedocs/gcc-4.9.3/gcc/SH-Options.html
 #
+#CFLAGS="-m4 -O2 -pipe"
+
 # If you set a CFLAGS above, then this line will set your default C++ flags to
 # the same settings.
 #CXXFLAGS="${CFLAGS}"
