--- make.conf	2006-10-16 17:06:32 +0100
+++ make.conf.sparc-fbsd	2006-10-16 17:09:22 +0100
@@ -23,6 +23,13 @@
 # Example:
 #USE="X gtk gnome -alsa"
 
+# Host Setting
+# ============
+#
+# The version number at the end has to correspond to the version of the
+# profile and of freebsd-lib package.
+CHOST="sparc64-gentoo-freebsd6.2"
+
 # Host and optimization settings 
 # ==============================
 #
@@ -61,7 +68,7 @@
 # DO NOT PUT ANYTHING BUT YOUR SPECIFIC ~ARCHITECTURE IN THE LIST.
 # IF YOU ARE UNSURE OF YOUR ARCH, OR THE IMPLICATIONS, DO NOT MODIFY THIS.
 #
-#ACCEPT_KEYWORDS="~arch"
+ACCEPT_KEYWORDS="~sparc-fbsd"
 
 # Portage Directories
 # ===================
